src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -396,7 +396,6 @@ private synchronized void freeResources(List<String> unlockResourceNames, @Nulla
 	public synchronized void unlock(List<LockableResource> resourcesToUnLock, @Nullable Run<?, ?> build) {
 		unlock(resourcesToUnLock, build, null, false);
 	}
-
```

Comment:
Why did this blank line between routines disappear? ;)

Reply:
Definitely a mistake on my end; missed it on my refactor